packages/form-core/src/utils/getAriaElementsInRightDomOrder.js

```diff
@@ -1,5 +1,12 @@
+/* eslint-disable no-bitwise */
 import { browserDetection } from '@lion/core';
 
+const moveDownConditions = [
+  Node.DOCUMENT_POSITION_PRECEDING,
+  Node.DOCUMENT_POSITION_CONTAINS,
+  Node.DOCUMENT_POSITION_CONTAINS | Node.DOCUMENT_POSITION_PRECEDING,
```

Comment:
Happened to notice this. Did you mean to use a single `|` ?

Reply:
jup that's a bitmaks operation (merge bytes e.g. the resulting number will represent a bitmask OR)... the `compareDocumentPosition` function is quite old is expected to be used with a bitmask